utils/tracing/integ-tests/client-server/helloworld.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
Why is this file new?

Reply:
It wasn't there yeah, and the generated proto file was just there. If someone later on looks at the generated proto file its not clear how it was generated, which is why I put it there.